### PR TITLE
Update Gatsby 2.24 to 4.10

### LINKED ISSUE #80: Gatsby out of date
The version of Gatsby in use is 2.24, latest is 4.10.

### PR DESCRIPTION
**Pull request by ericherman**
Fixes: #80

A review of _dependabot_ alerts called for an update of Gatsby.

Previously, `npm install` reported:

```
56 vulnerabilities (31 moderate, 24 high, 1 critical)
```

now `npm install`, reports:

```
28 vulnerabilities (6 moderate, 22 high)
```

This is far from ideal, yet an improvement.

### CODE REVIEW

**Review by Ainali — APPROVED** [review]
Yes, a step in the right direction!

**Comment by sergei-maertens** [comment]
isn't this just all build tooling that just generates a static website? I'd like to point out https://overreacted.io/npm-audit-broken-by-design/ again, even though it's a good feeling to have npm audit/dependabot stop complaining :)

File: .node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+v16.13.1
```
Comment (sergei-maertens) [inline .node-version:1]: how exactly does this file work? what's the impact?

I'm aware of `.nvmrc` that allows you to run `nvm use` in the root of the project to pick the appropriate node version, is this similar? which tooling picks up this file?

**Review by sergei-maertens — APPROVED** [review]

File: .node-version (hunk 1 above)
Comment (ericherman) [reply .node-version:1]: Working on a similar update on signalen.org, I discovered that the `.node-version` is read in the Netlify build-and-deploy process -- without which it was defaulting to a version of node too old for the updated Gatsby.

**Comment by ericherman** [comment]
Indeed, the end result is a static website, and while the static content includes JavaScript, the Dependabot reporting is largely safely ignorable.

Perhaps I shouldn't have mentioned the `npm audit` results, or giving more context; I included it for transparency that this does not "go green" .... though I suspect everyone who works with node is grumpy about the state of libraries and the reporting there.

The review of the Dependabot alerts highlighted that the central library is two major versions behind the current release, issue #80, thus it seems Dependabot serves some purpose, even when mostly "crying wolf" in a noisy fashion.

**Pull request merged**